packages/workspace/src/affine/login.ts

```diff
@@ -78,6 +80,32 @@ export const getLoginStorage = (): LoginResponse | null => {
   return null;
 };
 
+export const storageChangeSlot = new Slot();
+
+export const checkLoginStorage = async (
+  prefixUrl = '/'
+): Promise<LoginResponse> => {
+  const storage = getLoginStorage();
+  assertExists(storage, 'Login token is not set');
```

Comment:
I think it should be OK to have no login token here and just return undefined Login info

Reply:
i think it's not ok since the caller do need auth otherwise it will 401 anyway